.github/workflows/build-and-test-pipeline.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag todo-app:test .
     - run: docker run todo-app:test tests
-    - run: docker run -e CONNECTION_STRING="${{ secrets.DB_CONNECTION_STRING }}" -e SECRET_KEY=${{ secrets.SECRET_KEY }} todo-app:test e2e_tests
+    - run: docker run -e CONNECTION_STRING="${{ secrets.DB_CONNECTION_STRING }}" -e SECRET_KEY=${{ secrets.SECRET_KEY }} -e LOGIN_DISABLED=True todo-app:test e2e_tests
```

Comment:
Presumably this is unimportant since you set it directly in the tests, but no harm in having it here